### DISCUSSION TITLE
Setting cultivar parameters via config file

### DISCUSSION THREAD

**Discussion by BrianCollinss** [APSIMInitiative/ApsimX#9547] [Q&A] [answered]
Can I set a cultivar's parameter via a config file? How would I use the right path? Should I simply use the path to the parameter in the plant model tree?

**Comment by BrianCollinss** [comment]
Any help here please @par456 @ric394

**Reply by ric394** [reply, marked as answer]
You will want to change the 'Commands' parameter. So yes, path to the specific cultivar model you need to change and modify that parameter.

**Reply by BrianCollinss** [reply]
Thanks @ric394. What happens if the cultivar overwrites these parameters? Which one supersedes the other: cultivar parameter OR value from Config file?

**Reply by ric394** [reply]
The expected behavior is for the config file to supersede the cultivar parameter. As the value is overridden by the config file function.
